.travis.yml

```diff
@@ -1,16 +1,25 @@
 language: go
-go:
-- '1.13'
-services:
-- docker
-cache:
-  directories:
-  - "$GOPATH/pkg"
-before_install:
-- go get github.com/mattn/goveralls
-install:
-- make ci/install
-script:
-- make ci/test
-after_success:
-- goveralls -coverprofile=coverage.out -service=travis-ci
+
+go: '1.13'
+
+jobs:
+  include:
+    - stage: "Tests"
```

Comment:
Use `stage: test` instead? I think it's the more usual name, plus it matches `stage: deploy` better.